### PR TITLE
IBM Cloud snippets

### PR DESCRIPTION
**Pull request by HintikkaKimmo**
Added new IBM Cloud snippets based on the new IBM Cloud Terraform provider that covers SoftLayer + IBM PaaS + some of the SaaS product configs. More snippets to be added later stage

More at information on the new IBM Cloud Terraform provider at:  https://ibm-cloud.github.io/tf-ibm-docs/index.html

### CODE REVIEW

**Comment by rixrix** [comment]
hello @HintikkaKimmo thank you for your PR. I'll look into this today and push it to marketplace. cheers

**Comment by HintikkaKimmo** [comment]
Cool. I will start improving IBM snippets in a weeks time.

File: src/providers/ibmcloud.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+    "ibmcloud": {
+        "prefix": "tf-ibmcloud",
```
Comment (rixrix) [inline src/providers/ibmcloud.json]: tried on local machine, vscode wasn't able to provide intellisense for this - maybe it's my machine or the version of vscode. I'd suggest to change the  name and prefix to the following:
```
"ibm_provider": {
  "prefix": "tf-ibm_provider"...
}
```
that sample above seems to work on my machine.

my current setup:

vscode v1.19.2
on ubuntu Xenial

**Review by rixrix — COMMENTED** [review]
@HintikkaKimmo I found this very trivial issue. let me know your thoughts.

File: src/providers/ibmcloud.json (hunk 1 above)
Comment (HintikkaKimmo) [reply src/providers/ibmcloud.json]: Oke, I update it tomorrow.

Comment (rixrix) [reply src/providers/ibmcloud.json]: thanks @HintikkaKimmo I'll merge it right away and hopefully push it to marketplace when you get a chance to update your PR. cheers

**Comment by HintikkaKimmo** [comment]
Name changed to `tf-ibm_provider`

**Pull request merged**